lineman/app/components/thread_preview_collection/thread_preview_collection.scss

```diff
@@ -18,6 +18,7 @@
 .thread-preview__thread-header {
   @include fontCrapName;
   padding-bottom: 4px;
+  display: flex;
```

Comment:
it's magic @gdpelican. see https://css-tricks.com/snippets/css/a-guide-to-flexbox/

the page is long, but just enabling flex on a parent means you can have children in a row that sit next to each other nicely